### ISSUE TITLE
Loading the .m3u

### ISSUE THREAD

**Issue by quincycrystal** [iptv-ch/iptv-ch.github.io#47]
Hello,

I am relatively new to this IPTV game. So I wanted to ask you if there are any instructions on how to use these files? Can I just load the SwisscomIPTV-brut.m3u in VLC for example and watch the streams or do I need any subscriptions for that? 

Thank you in advance for your answer. 

Kind regards, 
quincy

**Comment by redge73** [comment]
> I need any subscriptions
Yes, and you need a decoder from Verimatrix for paid tv channels and don't know how to do it. I'm not a hacker at all. 
There so many iptv piracy solution for all paid tv channels do this job at your own risk.
This file "SwisscomIPTV-brut.m3u" is just a tracking list of all tv channels free/paid from Swisscom blue tv and not used in anyway for piracy in my case.
